res_partner_operating_unit/views/res_partner_view.xml

```diff
@@ -0,0 +1,41 @@
+<odoo>
+    <!-- Copyright 2017 Niaga Solution - Edi Santoso <repodevs@gmail.com>
+    Copyright (C) 2019 Serpent Consulting Services
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+
+    <!-- Partner Form View  -->
+    <record model="ir.ui.view" id="view_partner_form">
+    <field name="name">res.partner.form</field>
+    <field name="model">res.partner</field>
+    <field name="inherit_id" ref="base.view_partner_form"/>
+    <field name="arch" type="xml">
+        <field name="category_id" position="after">
+            <field name="operating_unit_ids" groups="operating_unit.group_multi_operating_unit" domain = "[('company_id','=', company_id)]" options="{'no_create': True}"/>
```

Comment:
```suggestion
            <field name="operating_unit_ids"
                   groups="operating_unit.group_multi_operating_unit"
                   domain = "[('company_id','=', company_id)]"
                   options="{'no_create': True}"
                   widget="many2many_tags"/>
```

Reply:
@Dangar-Serpentcs You are missing the widget...

Reply:
Improved